test/e2e/kuttl/ScaleUpAndDownDC/02-scaleDown-dc.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: db.orange.com/v1alpha1
+kind: CassandraCluster
+metadata:
+  name: cassandra-e2e
+spec:
+  nodesPerRacks: 1
```

Comment:
just update that one 👆 and remove the topology

Reply:
Updated :)